docs/pages/access-controls/guides/headless.mdx

```diff
@@ -32,8 +32,9 @@ For example:
 - A v12.2+ Teleport cluster with WebAuthn configured.
   See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
 - WebAuthn hardware device, such as YubiKey.
-- A Web browser with [WebAuthn support](
-  https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/).
+- Machines for Headless WebAuthn activities have [Linux](../../installation.mdx#linux), [Mac](../../installation.mdx#macos) or [Windows](../../installation.mdx#windows-tsh-client-only) `tsh` binary v12.2+ installed.
```

Comment:
Does does "activities" mean here?

Reply:
Any action that is being attempted with headless webauthn. Like tsh ssh, tsh scp with the --headless parameter